ui/components/component-library/select-wrapper/select-wrapper.stories.tsx

```diff
@@ -0,0 +1,311 @@
+import React from 'react';
+import { StoryFn, Meta } from '@storybook/react';
+import { SelectButton } from '../select-button';
+import { SelectOption } from '../select-option';
+import { Button } from '../button';
+import { Text } from '../text';
+import { BackgroundColor } from '../../../helpers/constants/design-system';
+import README from './README.mdx';
+
+import { SelectWrapper, useSelectContext } from '.';
+
+export default {
+  title: 'Components/ComponentLibrary/SelectWrapper',
+  component: SelectWrapper,
+  parameters: {
+    docs: {
+      page: README,
+      story: {
+        inline: false,
+      },
+    },
+  },
+  argTypes: {},
```

Comment:
Can we add all the argTypes here so we can see all the events fire correctly in the actions tab?